packages/js/src/utils.ts

```diff
@@ -53,7 +53,7 @@ export async function captureError(fn: () => Promise<any>): Promise<string> {
 }
 
 export function isTopLevelAccount(accountId: string): boolean {
-  return accountId.includes('.');
```

Comment:
Why are we making this change?

Reply:
The logic was wrong.  A top level account is one without an `.` not with one.